_data/presentations.yml

```diff
@@ -1,4 +1,4 @@
-- title: Impacts of HALEU on the Nuclear Feul Cycle
+- title: "Impacts of HALEU on the Nuclear Fuel Cycle"
   presenter: Amanda M. Bachmann
   event: NC State University Buuilding Future Faculty Program
```

Comment:
I just noticed an extra u in Building here too

Reply:
fixed!